payload_bay_micro.ino

```diff
@@ -4,9 +4,28 @@
 #include "altimeter.h"
 #include "Buffer.h"
 
+// How often to signal the radio, in Hertz
+const unsigned long RADIO_FREQ = 20;
+// The last time the altitude was transmitted, in us
+unsigned long lastRadioTime = 0;
+
 Altimeter alt;
 
+void sendToRadio(float altitude) {
```

Comment:
Do we have a packet design for the radio bay as well now? I would like that to be put in a table somewhere (even discord would be fine)